src/components/common/Search/Search.tsx

```diff
@@ -202,6 +209,43 @@ export class Search extends React.Component<SearchProps, SearchState> {
     );
   };
 
+  onKeyDown = (e: KeyboardEvent) => {
+    const { liveSearch, onKeyDown } = this.props;
+    const { cursor, liveSearchResults, liveSearchShow } = this.state;
+
+    if (onKeyDown) {
+      onKeyDown(e);
+    }
+    if (e.keyCode === 38 || e.keyCode === 40) {
+      e.preventDefault();
+    }
+
+    if (!liveSearch || !liveSearchShow) {
+      return;
+    }
+
+    // Arrow up
+    if (e.keyCode === 38 && cursor && cursor > 0) {
+      this.setState({ cursor: cursor - 1 });
```

Comment:
Not handle a loop? So if when `cursor===0` will result in `cursor=liveSearchResults.length - 1`

Reply:
fixed